packages/components/src/components/modal/modal.tsx

```diff
@@ -247,4 +252,11 @@ export class Modal {
     this.dismiss.emit();
     this.close.emit();
   };
+
+  private headerText = (): void => {
+    this.ariaLabelText =
+      this.heading ||
+      (hasNamedSlot(this.host, 'heading') && getSlotTextContent(this.host, 'heading')) ||
+      (hasNamedSlot(this.host, 'header') && getSlotTextContent(this.host, 'header'));
+  };
```

Comment:
```suggestion
  private getAriaLabel = (): string => 
      this.heading ||
      (hasNamedSlot(this.host, 'heading') && getSlotTextContent(this.host, 'heading')) ||
      (hasNamedSlot(this.host, 'header') && getSlotTextContent(this.host, 'header'));
```

Reply:
@henrijoss 
Had to change the return statement to long-term to match the SSR regex of stripping out private functions.